hoot-core-test/src/test/cpp/hoot/core/algorithms/AlphaShapeTest.cpp

```diff
@@ -147,3 +146,4 @@ class AlphaShapeTest : public CppUnit::TestFixture
 };
 
 CPPUNIT_TEST_SUITE_NAMED_REGISTRATION(AlphaShapeTest, "quick");
+}
```

Comment:
The `CPPUNIT_TEST_SUITE_NAMED_REGISTRATION` macros have to be inside of the namespace in order to work.